news-catalyst/next-tinynewsdemo
# Feature/fallback on category pages

Issue #153 

In order to dynamically serve a category/section page for a newly created section that didn't exist at build time, I had to switch the section template to use next's fallback feature. I also had to make a few other small changes:

1. related to the fallback feature, I added a check if fallback & display a very barebones "Loading..." message; will need design thought
2. instead of loading sections on this page from cache, load fresh from API instead; otherwise the fallback fails

To test:

* on a built & deployed app (local: run `yarn clear && yarn populate && yarn build && yarn start)...
* add a new category http://localhost:3000/tinycms/config/categories/add
* try loading the new category page http://localhost:3000/<categoryName>
* you should see "Loading..." briefly then the page should render
